themes/digital.gov/src/scss/new/_newsletter-signup.scss

```diff
@@ -1,181 +1,26 @@
 @use "uswds-core" as *;
 
-// Newsletter
+// GovDelivery Newsletter
 #newsletter-signup{
+  height: 22.6875rem;
```

Comment:
Let's add the height on desktop only, there's a visual quirk in mobile.

![image](https://user-images.githubusercontent.com/3385219/197801711-331f8c6f-1a3e-4abb-b161-4a5c6f7a50c7.png)

Reply:
Thanks Nick!